101-webapp-basic-windows/azuredeploy.json

```diff
@@ -55,12 +49,13 @@
     "variables": {
         "appServicePlanPortalName": "[concat('AppServicePlan-', parameters('webAppName'))]",
         "gitRepoReference":{
+            "none": "",
             ".net":"https://github.com/Azure-Samples/app-service-web-dotnet-get-started",
             "node":"https://github.com/Azure-Samples/nodejs-docs-hello-world",
             "php":"https://github.com/Azure-Samples/php-docs-hello-world",
             "html":"https://github.com/Azure-Samples/html-docs-hello-world"
         },
-        "gitRepoUrl":"[if ( bool(parameters('sample')), variables('gitRepoReference')[parameters('language')], parameters('repoUrl') ) ]",
+        "gitRepoUrl":"[if(empty(parameters('repoUrl')), variables('gitRepoReference')[parameters('language')], parameters('repoUrl'))]",
```

Comment:
@bmoore-msft  The scenario of creating an empty PHP, Node, and HTML app is missed if we do it this way. That is not a requirement for Docs, but was doing it for template flexibility.

If there's further concerns, can we do it over a brief call on teams? I'd like to see if we can push the sample and Docs article this week. Thanks!